### ISSUE TITLE
task_sequence_labeling_ner_crf 运行loss为负值

### ISSUE THREAD

**Issue by uRENu** [bojone/bert4keras#334]
### 基本信息

参考您示例中的task_sequence_labeling_ner_crf.py进行ner任务，数据是msra的ner数据（数据共有28145条句子，将其中的90%条句子用于训练），预训练模型用的是nezha-wwm-base，但是在训练过程中loss在第一个epoch就从80左右降到了-39.8206，这个loss值看起来很不正常，已经检查过输入的数据是没问题的。
我看了您在crf中的loss函数：
 def dense_loss(self, y_true, y_pred):
       """y_true需要是one hot形式
        """
        # 导出mask并转换数据类型
        mask = K.all(K.greater(y_pred, -1e6), axis=2, keepdims=True)
        mask = K.cast(mask, K.floatx())
        # 计算目标分数
        y_true, y_pred = y_true * mask, y_pred * mask
        target_score = self.target_score(y_true, y_pred)
        # 递归计算log Z
        init_states = [y_pred[:, 0]]
        y_pred = K.concatenate([y_pred, mask], axis=2)
        input_length = K.int_shape(y_pred[:, 1:])[1]
        log_norm, _, _ = K.rnn(
            self.log_norm_step,
            y_pred[:, 1:],
            init_states,
            input_length=input_length
        )  # 最后一步的log Z向量
        log_norm = tf.reduce_logsumexp(log_norm, 1)  # logsumexp得标量
        # 计算损失 -log p
        return log_norm - target_score

loss不应该是直接返回log_norm吗，想问下这里为什么要减去转移得分 target_score？

- 环境：
python3.6、tf1.15-gpu、keras2.3.1、

### 参数设置
bert_layers = 12  # 1-12 ，越小越靠前，相应的learning rate 也要相应调大
maxlen = 150
epochs = 15
batch_size = 32
crf_lr_multiplier = 1000  # 扩大CRF层的学习率
learning_rate = 1e-5
grad_accum_steps = 2  # 梯度积累，模拟batch_size=32

loss=CRF.sparse_loss

### 自我尝试
我试过更改crf_lr_multiplier 但是影响不大，还是会在第一个epoch时，loss就会降到负值
